config/packages/security.yaml

```diff
@@ -22,7 +48,7 @@ security:
     # Easy way to control access for large sections of your site
     # Note: Only the *first* access control that matches will be used
     access_control:
-        # - { path: ^/admin, roles: ROLE_ADMIN }
+        - { path: ^/admin, roles: ROLE_ADMIN }
```

Comment:
Wellicht nog een expliciete config opnemen voor de /saml/metadata route?

Reply:
Op een later moment opnemen en testen of dit werkt bij de saml_based login